index.html

```diff
@@ -0,0 +1,198 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/39329175e7.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="./reset.css">
+    <link rel="stylesheet" href="./style.css">
+    <title>Nova</title>
+</head>
+
+<body>
+    <header class="header-background">
+        <section class="header container">
+            <img class="header__logo" src="./images/logo.svg" alt="nova logo">
+            <nav class="nav">
+                <ul class="nav__items">
+                    <li class="nav__item"> <a class="nav__link" href="#team">The Team</a></li>
+                    <li class="nav__item"> <a class="nav__link" href="#pricing">Pricing</a> </li>
+                    <li class="nav__item"> <a class="nav__link" href="#features">Features</a> </li>
+                </ul>
+            </nav>
+        </section>
+        <section class="hero container">
+            <div class="hero__text-box">
+                <h1 class="hero__heading">Beautiful Free Nova template</h1>
+                <p class="hero__description">A top notch premium quality template for your next web project.</p>
+                <a href="#" class="hero__btn">Download <span>free</span>!</a>
+            </div>
+            <div class="hero__img-box">
+                <img class="hero__img" src="./images/screen.png" alt="hero image">
+            </div>
+        </section>
+    </header>
+
+    <main>
+        <section class="features" id="features">
+            <h2 class="features__header container">Features</h2>
+            <div class="features-box container">
+                <ul class="feature__item">
+                    <li class="feature__icon"><img src="./images/responsive.svg" alt="mobile phone logo"></li>
+                    <li class="feature__name">fully responsive</li>
+                    <li class="feature__description">Looks amazing on any device: smartphone, tablet, laptop and
+                        desktop.</li>
+                </ul>
```

Comment:
W zasadzie zrobiłbym inaczej. Każdy z tych elementów to lista "cech" dlatego:
```
<ul>
  <li>
    cecha 1
  </li>
  <li>
    cecha 2
  </li>
  ...
</ul>
```

Następnie każda cecha składa się z:
- zdjęcia
- tekst
- opis

Te elementy osadziłbym  w poszczególnych znacznikach - nie koniecznie `ul > li` tylko:
- `<h3>` - tytuł
- `<p>` - opis
-  `<figure>` > `img` - zdjęcie